gateway/src/apicast/policy/soap/init.lua

```diff
@@ -0,0 +1 @@
+return require('soap')
```

Comment:
So I started to think about this. And I think we can expose some support tooling in different module, so it can be unit tested.

Lets say the main policy code is in `soap_policy.lua`. Then there can be `soap.lua` that has stuff like "extracting the soap action" and it can be unit tested in busted. 

That would allow us not exposing extra methods on the policy, but still exposing it internally (if the loading works) for tests. And policies should not be able to load other policies (but that is not enforced yet), so we should be fine and the code should be used only from tests.

Just some food for though. I'd like to hear your take.

Reply:
That would be a nice improvement.